cmd/monitorcodegen/main.go

```diff
@@ -72,11 +72,9 @@ func generate(templateFile string, force bool) error {
 	tmpl, err := template.New(generatedMetadataTemplate).Option("missingkey=error").Funcs(template.FuncMap{
 		"formatVariable": func(s string) (string, error) {
 			formatted, err := formatVariable(s)
-
```

Comment:
Unnecessary removal of whitespace, same thing below.

Reply:
ok